packages/x-components/.eslintrc.js

```diff
@@ -6,8 +6,8 @@ module.exports = {
   },
   rules: {
     'no-dupe-class-members': 'off',
-    '@typescript-eslint/no-unused-vars': 'off',
-    'vuejs-accessibility/alt-text': 'warn',
+    '@typescript-eslint/no-unused-vars-experimental': 'off',
+    'vuejs-accessibility/alt-text': 'error',
```

Comment:
Should we set this here, or should we set it in the eslint-plugin-x? 

I know this will make all the projects that use the plugin to update the alt images, but i am concerned about enabling this rule in every single project one by one.

Reply:
Actually `'vuejs-accessibility/alt-text': 'error',` should be removed from here as the recommended accessibility configuration (which is the same as listed here but with `'error'`s instead of `'warn'`s was already added to `packages/eslint-plugin-x/lib/configs/vue.js`, so what we are actually doing here is to avoid the build breaks due to code not (yet) fulfilling the rules